tests/Golden.hs

```diff
@@ -13,7 +13,7 @@ import Data.Foldable (for_)
 import Data.Text.Lazy (Text)
```

Comment:
I ran the tests, expecting to only see small differences due to the pretty-printing now using `∀(α : *)` instead of `∀α`, but the test output was so unreadable that I couldn't even tell if those were the differences or not! So I spent a bit of time improving the output format in the failure case.

Reply:
That's wonderful! Thank you!